docker/entrypoint.sh

```diff
@@ -44,6 +44,15 @@ function ensure_required_variables {
   : "${AUTH_LDAP_GROUPS_MEMBER:? AUTH_LDAP_GROUPS_MEMBER required}"
 }
 
+function copy_clients_certificates {
```

Comment:
```suggestion
function copy_client_configurations_directory {
```

Nit: CCD stands for "Client Configuration Directory"